targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC/analogin_api.c

```diff
@@ -70,7 +80,7 @@ uint16_t analogin_read_u16(analogin_t *obj)
 
     adcConvSeqConfigStruct.channelMask = (1U << channel);
     adcConvSeqConfigStruct.triggerMask = 0U;
-    adcConvSeqConfigStruct.triggerPolarity = kADC_TriggerPolarityNegativeEdge;
+    adcConvSeqConfigStruct.triggerPolarity = kADC_TriggerPolarityPositiveEdge;
```

Comment:
Any partricular reason why this polarity flip was needed?

Reply:
This is to match the values set inside the MCUXpresso SDK examples.